src/AssignmentThree.java

```diff
@@ -60,6 +89,15 @@ public Card(char value, Suit suit)
       set(value, suit);
    }
 
+   /**
+    * 
+    * @param card
+    * @return Copy of Card
+    */
+   public static Card newInstance(Card card) {
```

Comment:
While I love the idea of a copy method, I think we should stick to the spec so that we don't lose any potential points. See comment below where this method was called.